### PR TITLE
Update tokens with latest design and fix some component colors

### PR DESCRIPTION
**Pull request by djalmaaraujo**
## Description

Updates for the dark mode theme needed after some design reviews.

### CODE REVIEW

File: src/system/Form/RadioBoxGroup.js [hunk 1]
```diff
@@ -33,31 +33,42 @@ const RadioBoxGroup = React.forwardRef(
 					id={ `o${ index }` }
 					sx={ {
 						p: 3,
-						background: 'none',
```
Comment (andrea-sdl) [inline src/system/Form/RadioBoxGroup.js:36]: Is this change working as expected? 
They now look gray in the light theme.
[Deploy preview link](https://deploy-preview-167--vip-design-system-components.netlify.app/?path=/story/radioboxgroup--default)

Screenshot RadioBoxGroup - Default ⋅ Storybook 2023-01-23 11-57-47

**Review by andrea-sdl — COMMENTED** [review]
@djalmaaraujo left a minor comment on the RadioBoxGroup, rest is working fine.

File: src/system/Form/RadioBoxGroup.js (hunk 1 above)
Comment (djalmaaraujo) [reply src/system/Form/RadioBoxGroup.js:36]: @andrea-sdl Check again ;)

**Comment by djalmaaraujo** [comment]
@andrea-sdl Merging since this is going to beta already in 0.28.5. I can fix anything else. I want to avoid any commits in trunk

**Pull request merged**